### ISSUE TITLE
GitHub Actions: Replace usage of set-output due to deprecation

### ISSUE THREAD

**Issue by batbattur** [chuhlomin/render-template#7]
## Problem

GitHub announced they are deprecating `set-output` instead of waiting until they fully disable the feature: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

It looks like it's only used in here in the action (besides the `release.yml` workflow): https://github.com/chuhlomin/render-template/blob/0abed2ba8fdefa561641617741a1e879f497223b/main.go#L59

The proposed change is to append the `result` to GITHUB_OUTPUT env variable:

```
if os.Getenv("GITHUB_OUTPUT") != "" {
	os.Setenv("GITHUB_OUTPUT", fmt.Sprintln(os.Getenv("GITHUB_OUTPUT"))+fmt.Sprintf("result=%s", escape(output)))
} else {
	os.Setenv("GITHUB_OUTPUT", fmt.Sprintf("result=%s", escape(output)))
}
```

**Comment by chuhlomin** [comment]
Hey, apparently, env var `GITHUB_OUTPUT` contains a path to the file,
like this `/github/file_commands/set_output_d8edcdd6-9419-4972-a006-8e6326a1f1e0`.

Just released [version 1.6](https://github.com/chuhlomin/render-template/releases/tag/v1.6), give it a try

**Comment by peter-evans** [comment]
@chuhlomin I can confirm that v1.6 works well and the deprecation warning is gone. Thank you.
